lib/github/api.nit

```diff
@@ -621,6 +621,46 @@ class Commit
 
 	# Commit message.
 	var message: nullable String is writable
+
+	# Git commit representation linked to this commit.
+	var commit: nullable GitCommit
+end
+
+# A Git Commit representation
+class GitCommit
+	super GithubEntity
+	serialize
+
+	# Commit SHA.
+	var sha: nullable String is writable
+
+	# Parent commits of `self`.
+	var parents: nullable Array[GitCommit] = null is writable
+
+	# Author of the commit.
+	var author: nullable GitUser is writable
+
+	# Committer of the commit.
+	var committer: nullable GitUser is writable
+
+	# Commit message.
+	var message: nullable String is writable
+end
+
+# Git user authoring data
+class GitUser
+	super GithubEntity
+	serialize
+
+	# Authoring date.
+	var date: nullable String = null is writable
```

Comment:
Yes, when Github doesn't know the account associated to the commiter/author, it create a special kind of user with the signed-off authoring date.

The modelisation is a little sketchy to accommodate the type constraints in the deserializer.